CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 ## Unreleased
 
 ### Added
+* **Breaking** Added content share metrics as new enums in `ObservableMetric`.
+* Added content share APIs that supports a 2nd video sending stream such as screen capture, read [content share guide](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/content_share.md) for details.
+* Added in app only screen share to demo app.
+* Added `AmazonChimeSDKDemoBroadcast` Broadcast Upload Extension to the demo app to share device level screen.
+* Added message for video tiles paused by poor network in demo app.
 * Update demo application to pause/resume remote videos and stop/start local video when app is in background/foreground
 
 ### Fixed
 * Fix the demo application bug that front camera video was not mirrored without selecting video device on the video preview page.
 
-## 0.13.0 - 2020-12-17
+### Changed
+* **Breaking** `AudioVideoFacade` now also implements `ContentShareController`.
```

Comment:
Why is this breaking?

Reply:
Because if any builder is implemeting AudioVideFacade, they will have to implement ContentShareController APIs